src/plugins/lightbox/_base.scss

```diff
@@ -11,6 +11,10 @@
 	outline-offset: -2px;
 }
 
+%lightbox-modal-hide {
+	visibility: hidden!important;
```

Comment:
According to which SCSS Lint rule? if the build isn't complaining about it, should we add something to the rule for it?